packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js

```diff
@@ -282,6 +286,12 @@ export function createResponseState(
           stringToChunk(escapeTextForBrowser(integrity)),
         );
       }
+      if (crossOrigin) {
```

Comment:
This would fail for `crossOrigin: ""`. Should be something like `if (typeof crossOrigin === 'string')`